<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_4</class>
 <widget class="QDialog" name="fct_comp_4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sportifs d'un pays et un Ã©quipe</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>342px-Olympic_rings_without_rims.svg.png</normaloff>342px-Olympic_rings_without_rims.svg.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pays</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_fct_4_pays"/>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Equipe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_fct_4_equipe"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table_fct_comp_4">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>70</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>numSp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nomSp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>prenomSp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>categorieSp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>dateNaisSp</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_fct_comp_4">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_fct_4_pays</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_4</receiver>
   <slot>initialiserid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_fct_4_equipe</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_4</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
  <slot>initialiserpays()</slot>
  <slot>initialiserid()</slot>
 </slots>
</ui>
